usage.txt

```diff
@@ -15,6 +15,7 @@ app-bundle-id      bundle identifier to use in the app plist (darwin platform on
 app-version        release version to set for the app
 asar               packages the source code within your app into an archive
 asar-unpack        unpacks the files to app.asar.unpacked directory whose filenames regex .match this string
+asar-unpack-dir    unpacks the dirs to app.asar.unpacked directory whose names match this string
```

Comment:
Not totally. The value of `asar-unpack-dir` not need to match the beginning of the absolute file path of `sourcedir`. It assumes it is under the `sourcedir`.
For example:
```
electron-packager /your/path/to/app app_name --asar-unpack-dir=dir1
```
This will unpack the `/your/path/to/app/dir1` directory.